### PR TITLE
PhpdocIndentFixer - fix edge case with inline phpdoc

### LINKED ISSUE #2844: Bug when running braces fixer along side phpdoc_indent
On current master branch, consider the following input:

```php
<?php

function foo()
{
    $foo->bar(/** oops */$baz);
}
```

When, either the `braces` fixer or the `phpdoc_indent` fixer are run separately, by themselves, no fixes are made, however, when they are run together, we crash:

```
UnexpectedValueException Missing block start. 
    .../Tokenizer/Tokens.php:469 PhpCsFixer\Tokenizer\Tokens::findBlockEnd
    .../Fixer/Basic/BracesFixer.php:345 PhpCsFixer\Fixer\Basic\BracesFixer::fixIndents
    .../Fixer/Basic/BracesFixer.php:147 PhpCsFixer\Fixer\Basic\BracesFixer::applyFix
```

image

image

### PR DESCRIPTION
**Pull request by keradus**
Replaces #2847 
Closes #2844
Closes #2845

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
👍 (though, fix does have the wrong commit author :trollface:)

File: tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php [hunk 1]
```diff
@@ -381,6 +381,16 @@ public function bar()
...
 
+        $cases[] = array(
+            '<?php
+
+function foo()
+{
+    $foo->bar(/** oops */$baz);
+}
+',
+            );
```
Comment (julienfalque) [inline tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php]: Wrong indentation.

File: tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php [hunk 2]
```diff
@@ -381,6 +381,16 @@ public function bar()
 ',
         );
 
+        $cases[] = array(
+            '<?php
+
+function foo()
+{
+    $foo->bar(/** oops */$baz);
```
Comment (SpacePossum) [inline tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php]: wonder what happens with
```php
 $foo->bar($a,/** oops */$baz);
```
?

Comment (GrahamCampbell) [inline tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php]: Works just fine. I tested that earlier. Probably worth adding it as a unit test though. 👍

File: tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php (hunk 1 above)
Comment (keradus) [inline tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php]: nice catch

File: tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php (hunk 2 above)
Comment (keradus) [inline tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php]: added

Comment (keradus) [inline tests/Fixer/Phpdoc/PhpdocIndentFixerTest.php]: actually it didnt... added, fixed

**Pull request closed without merging**